### ISSUE TITLE
Ability to Ask Indigo for device status

### ISSUE THREAD

**Issue by msreynolds** [msreynolds/askIndigo#5]
As a user, I would like to be able to Ask Indigo for the status of any given device

**Comment by foriegnbody** [comment]
Hey there - this is really good stuff! I can see it wants to contact my Indigo server but the lamdba function is getting an error "Unexpected exception Error: Cannot find module 'request-digest'"

Any thoughts? I don't see in the src tree like a request-digest package so I presume it already exists in aws's node js support? Do you think they updated the version number (I see the package.json has it as "0.0.9").

Or, am I being a tool and do I need to do something else like an npm install into the src directory? (I don't have npm currently on my mac but..)

Any help you could throw my way would be really appreciated!

Thanks,
Ian

**Comment by msreynolds** [comment]
@foriegnbody Yeah you need to do an npm install so that the request-digest module gets included in the zip file you upload to Amazon.  The README.md has all the info you need.

**Comment by foriegnbody** [comment]
Yeah I figured that out and couldn't find me comment to update!

Worked great but I made a few tweaks to better handle errors from indigo
not finding a device and to put it into conversation mode (so you can
adjust more than one light without having to 'ask indigo' each time.
Still playing but it's a great effort man - it's pretty damn seamless!

If I make further changes I'm happy to throw a pull request and you can
merge whatever you like. Traveling to UK next week but I'm sure I'll play
more when I'm back.

Ian

**Comment by msreynolds** [comment]
@foriegnbody Cool!  I look forward to any PR's you might make, there are lots of things I would like to improve on this project.  I haven't updated anything since the latest portal changes came through in the last month...
